user/TASK/chassis_task/chassis_task.c

```diff
@@ -11,11 +11,209 @@
 #include "CAN_Receive.h"
 #include "delay.h"
 #include "USART_comms.h"
+#include "INS_task.h"
+#include "remote_control.h"
+#include "pid.h"
 
 
+static uint8_t chassis_init(Chassis_t *chassis_init);
+static void chassis_update_data(Chassis_t *chassis_update);
+static void chassis_set_mode(void);
+static void chassis_remote_calc(chassis_user_mode_e mode);
+static void chassis_set_mode(void);
+static void chassis_motor_calc(void);
+static void chassis_PID(void);
+
+
+static Chassis_t chassis;
+    
+fp32 def_pid_constants[3] = {0.02, 0.0, 0.1};
```

Comment:
Once we implement more pid, it may be better to put these in the pid file. I would leave these here while we are still testing and tuning though